### PR TITLE
fix: Local iso upload in content library item

### PR DESCRIPTION
**Pull request by juanvalino**
### Description

This fixes local iso uploading in content library item.

### Acceptance tests
- [ ] Have you added an acceptance test for the functionality being added?
- [ ] Have you run the acceptance tests on this branch?

### Release Note
Release note for [CHANGELOG](https://github.com/hashicorp/terraform-provider-vsphere/blob/main/CHANGELOG.md):

```release-note
Fix content library item local iso upload
```
### References

Resolves https://github.com/hashicorp/terraform-provider-vsphere/issues/1664

### CODE REVIEW

**Comment by tenthirtyam** [comment]
Hi @juanvalino, 

Thanks for the contribution. The GHA tests are failing for the following:

```console
gofmt needs running on the following files:
8
./vsphere/internal/helper/contentlibrary/content_library_helper.go
9
You can use the command: `make fmt` to reformat code.
10
make: *** [GNUmakefile:46: fmtcheck] Error 1
11
Error: Process completed with exit code 2.
```

Could you please run `go fmt` against the changed file and commit? That should take care of the issue.

**Ryan Johnson**
Staff II Solutions Architect | VMware, Inc.

**Comment by juanvalino** [comment]
@tenthirtyam I fixed the code format and now the tests seem to pass.

File: vsphere/internal/helper/contentlibrary/content_library_helper.go
Comment (tenthirtyam) [inline vsphere/internal/helper/contentlibrary/content_library_helper.go:204]: I'm not sure I understand why `isOva = false` is used here if it's already defined as false.

Comment (juanvalino) [reply vsphere/internal/helper/contentlibrary/content_library_helper.go]: You're right, it's unnecessary. I'm pushing a new version with this fixed.

**Review by tenthirtyam — APPROVED** [review]
Confirmed that this fix addresses the issue and works quite nicely. Tested with multiple ISOs being uploaded at the same time w/o any issue.

LGTM! 🚀

**Comment by tenthirtyam** [comment]
Hi @juanvalino, 

I've tagged the pull request for the v2.2.0 milestone and provider my review as a project collaborator. It still needs a review from the project maintainer ( @appilon ) before it can be merged.

**Ryan Johnson**
Staff II Solutions Architect | VMware, Inc.

**Review by appilon — APPROVED** [review]

**Comment by tenthirtyam** [comment]
Thanks, Alex!

**Ryan Johnson**
Staff II Solutions Architect | VMware, Inc.

**Pull request merged**
